modules/images-managing-images-enabling-imagestreams-kube.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/managing-images/using-imagestreams-with-kube-resources.adoc
+
+
+[id="images-managing-images-enabling-imagestreams-kube_{context}"]
+= Enabling imagestreams with Kubernetes resources
+
+When enabled, you can put a reference to an imagestream in the
+`image` field of a resource. When using this feature, you can only
+reference imagestreams that reside in the same project as the resource. The
+imagestream reference must consist of a single segment value, for example
+`ruby:2.5`, where `ruby` is the name of an imagestream that has a tag named
+`2.5` and resides in the same project as the resource making the reference.
+
+[NOTE]
```

Comment:
@bparees here's the PR as promised. Is this were you were looking to put the note? (Or at least close?)

Just a note, we're on Merge freeze until Monday.

Reply:
this spot for the note is fine